cloud/azure/bin/deploy

```diff
@@ -4,6 +4,14 @@ set -e
 source "cloud/shared/bin/lib/health.sh"
 source "cloud/azure/bin/lib.sh"
 
+while getopts t: flag
+do
+  case "${flag}" in
+    t) docker_tag=${OPTARG};;
```

Comment:
suggestion (blocking): flag name should be `--tag` to match https://github.com/seattle-uat/civiform-tf-deploy/blob/document-variables/bin/deploy#L3